python_modules/libraries/dagster-docker/dagster_docker/container_context.py

```diff
@@ -0,0 +1,108 @@
+from typing import TYPE_CHECKING, Any, Dict, List, NamedTuple, Optional
+
+from dagster import check
+from dagster.config.validate import process_config
+from dagster.core.errors import DagsterInvalidConfigError
+from dagster.core.storage.pipeline_run import PipelineRun
+
+from .utils import DOCKER_CONTAINER_CONTEXT_SCHEMA
+
+if TYPE_CHECKING:
+    from . import DockerRunLauncher
+
+
+class DockerContainerContext(
```

Comment:
docblock somewhere for this  - mostly just to capture current intent behind object